test/post-911-gib-status/actions/post-911-gib-status.unit.spec.js

```diff
@@ -0,0 +1,171 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { getEnrollmentData } from '../../../src/js/post-911-gib-status/actions/post-911-gib-status';
+import {
+  BACKEND_SERVICE_ERROR,
+  GET_ENROLLMENT_DATA_FAILURE,
+  GET_ENROLLMENT_DATA_SUCCESS
+} from '../../../src/js/post-911-gib-status/utils/constants';
+
+let oldFetch;
+let oldSessionStorage;
+let oldWindow;
+const setup = () => {
+  oldSessionStorage = global.sessionStorage;
+  oldFetch = global.fetch;
+  oldWindow = global.window;
+  global.sessionStorage = { userToken: '123abc' };
+  global.fetch = sinon.stub();
+  global.fetch.returns(Promise.resolve({
+    headers: { get: () => 'application/json' },
+    ok: true,
+    json: () => Promise.resolve({})
+  }));
+  global.window.dataLayer = [];
+  global.window.URL = {
+    createObjectURL: () => { },
+    revokeObjectURL: () => { }
+  };
```

Comment:
I'm really lamenting not being able to do this globally, but I'm not going down _that_ rabbit hole again so soon!